pkg/reddit/fetch.go

```diff
@@ -304,10 +304,10 @@ func getPost(postUrl string, root map[string]interface{}) (fetchResult interface
 				Duration:      int64(duration),
 				Type:          FetchResultMediaTypeVideo,
 			}, nil
-		case "rich:video": // files hosted other than reddit; This bot currently supports gfycat.com
+		case "rich:video": // files hosted other than reddit; This bot currently supports Gfycat.com
 			if urlObject, domainExists := root["domain"]; domainExists {
 				switch urlObject.(string) {
-				case "gfycat.com": // just act like gif
+				case "Gfycat.com": // just act like gif
```

Comment:
I think this should not be like this. The domain must be lowercase

Reply:
It should work either way, but I agree. I did a find-and-replace, and that’s how that got there. (: I’ll change this back on my side.